src/view/segmentation.js

```diff
@@ -27,22 +29,23 @@ export const handleSegmentation = prompts => {
 		prompts.forEach( prompt => {
 			const promptId = prompt.getAttribute( 'id' );
 			const isOverlay = prompt.classList.contains( 'newspack-lightbox' );
+			const override =
+				// eslint-disable-next-line no-nested-ternary
+				getRawId( promptId ) === getPreviewedPromptId() // If previewing a single prompt, it should always be displayed.
+					? true
+					: isOverlay && overlayDisplayed // If an overlay and another overlay has already been displayed, it should not be displaeyd.
+					? false
+					: null; // Default behavior lets frequency/segmentation determine whether it should be dipslayed.
```

Comment:
This logic is getting a bit more difficult to read. Perhaps it can move to a separate function with guard clauses?